jubilant/_test_helpers.py

```diff
@@ -0,0 +1,27 @@
+import contextlib
+import secrets
+from collections.abc import Generator
+
+from ._juju import Juju
+
+
+@contextlib.contextmanager
+def temp_model(keep: bool = False) -> Generator[Juju, None, None]:
+    """Context manager to create a temporary model for running tests in.
+
+    This creates a new model with a random name in the format ``jubilant-abcd1234``, and destroys
+    it and its storage when the context manager exits.
+
+    Provides a :class:`Juju` instance to operate on.
+
+    Args:
+        keep: If true, keep the created model around when the context manager exits.
+    """
+    juju = Juju()
+    model = 'jubilant-' + secrets.token_hex(4)  # 4 bytes (8 hex digits) should be plenty
+    juju.add_model(model)
+    try:
+        yield juju
+    finally:
+        if not keep:
+            juju.destroy_model(model, destroy_storage=True, force=True)
```

Comment:
Does this need to have force? Even if the model is in a bad state it should still be able to be non-force destroyed in normal circumstances, right? What's the Juju'er stance on this?

Reply:
Yeah, I'm unsure about this. But my thinking was that because this is used in test cleanup, we should do whatever we can to ensure it cleans up after itself, so `--force` seems appropriate here.